### PR TITLE
Styling/sass architecture

### PR DESCRIPTION
**Pull request by monshan**
## What does this PR do (summary of changes)?

 - Applies gradient and font-family to App Component and specific children where necessary
 
## How should it be tested?

 - User can observe dynamic changes to the nav bar and cards as they navigate the application

## Future Iterations:

 - Modify for responsiveness

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -58,13 +59,12 @@ class App extends Component {
       <div className="App">
         <Route exact path="/">
           <section className="home-container">
-            <h1 className="home-title">CarryOkay</h1>
-            <p className="home-greeting">Hello friend 😬</p>
+            <h1 className="home-title glow">CarryOkay</h1>
+            <p className="home-greeting">Ready to sing?</p>
```
Comment (elizhahn) [inline src/App.js:63]: yay I like this change!

Comment (m1073496) [inline src/App.js:63]: Nice (-:

**Comment by m1073496** [comment]
[Styling](https://trello.com/c/FtdIqOEJ/91-styling)

File: src/components/Navigation/Navigation.scss [hunk 2]
```diff
@@ -43,7 +43,7 @@
 
 .home-my-songs { 
   border-radius: 147px 0 0 147px;  
-  background-color: rgba(202, 17, 156, 0.5); 
+  // background-color: rgba(202, 17, 156, 0.5);
```
Comment (elizhahn) [inline src/components/Navigation/Navigation.scss:46]: should we remove this

File: src/App.scss [hunk 3]
```diff
@@ -13,26 +15,41 @@ body {
...
+  0 0 40px rgb(252, 22, 225),
+  0 0 80px rgb(252, 22, 225),
+  0 0 90px rgb(252, 22, 225),
+  0 0 100px rgb(252, 22, 225),
+  0 0 150px rgb(252, 22, 225);
 }
 
 .home-greeting {
-  font-size: calc(10px + 2vmin); 
+  // font-size: calc(10px + 2vmin);
```
Comment (elizhahn) [inline src/App.scss:51]: should this be removed?

File: src/components/Navigation/Navigation.scss [hunk 4]
```diff
@@ -66,6 +66,18 @@
 }
 
 .off-home-nav {
+  height: 100px;
   display: grid;
-  grid-template-columns: auto 125px auto;
+  grid-template-columns: auto 160px auto;
+  // background: 
+  //   url('./francesco-de-tommaso-gskWuDulJn8-unsplash.jpg') no-repeat,
+  //   linear-gradient( 120deg, #a6c0fe 0%, #f68084 100%);
```
Comment (elizhahn) [inline src/components/Navigation/Navigation.scss:74]: remove?

**Review by elizhahn — APPROVED** [review]

**Pull request closed without merging**